### DISCUSSION TITLE
[Polkadot] Different Address derived from same secret phrase.

### DISCUSSION THREAD

**Discussion by taskooh** [trustwallet/wallet-core#1281] [Q&A] [answered]
**Describe the bug**
When importing Polkadot account, different address is derived from same secret phrase compared to subkey.
**To Reproduce**
1. Run subkey(https://substrate.dev/docs/en/knowledgebase/integrate/subkey), and generate secret phrase and address.
```
% subkey generate --network polkadot
Secret phrase `napkin protect market echo rent lawn purpose brief decline device season shell` is account:
  Secret seed:      0xeae090bfa7231482606fb30758eeddaec67ee99c3228be61f9a1be82216e086c
  Public key (hex): 0x28fc375edd89c6eb9f3683ae03cd023184f4867a7895b47298b5c98221a2c125
  Account ID:       0x28fc375edd89c6eb9f3683ae03cd023184f4867a7895b47298b5c98221a2c125
  SS58 Address:     1vjqgKwXEowW1Ya54nfSVGaoYEvtHGH7JenrzykczgwpwXJ
```

copy secret phrase `napkin protect market echo rent lawn purpose brief decline device season shell` and import polkadot wallet into Trust wallet from this secret phrase. 
Created address was:`13E7KDY5TA8RTUwEwpSDFbgMdfb4n2jLHegjdgRNWU42SDdk`

**Expected behavior**
Address in Trust Wallet should be same as address in subkey. In this case , it should be `1vjqgKwXEowW1Ya54nfSVGaoYEvtHGH7JenrzykczgwpwXJ`.
**Screenshots**

**Additional context**
I'm not using this secret phrase as wallet.

**Comment by hewigovens** [comment, marked as answer]
This is a known issue, wallet core uses ed25519 for polkadot, subkey uses sr25519 and different derive paths

**Comment by Tur1ngM4ch1n3** [comment]
hi there, is there a way to have the correct derivation path used for substrate wallet?
It would help a lot of ppl with funds locked on wallets they can't control or restore without the path.
Thank you!
